.github/workflows/satis-update.yml

```
@@ -25,7 +25,7 @@ jobs:
         uses: peter-evans/repository-dispatch@v1
         with:
           token: ${{ secrets.WEBHOOK_TOKEN }}
-          repository: bluehost/satis
+          repository: newfold-labs/satis
           event-type: 'Trigger Satis Build'
```

@circlecube The event type should be changed: 
```suggestion
          event-type: 'satis-build'
```

See https://github.com/newfold-labs/wp-module-coming-soon/blob/trunk/.github/workflows/satis-update.yml#L29